openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Users.spec.ts

```diff
@@ -190,6 +190,9 @@ test.describe('User with Admin Roles', () => {
     await redirectToHomePage(adminPage);
     await settingClick(adminPage, GlobalSettingOptions.USERS);
     await adminPage.waitForLoadState('networkidle');
+    await adminPage.waitForSelector('.user-list-table [data-testid="loader"]', {
+      state: 'detached',
+    });
```

Comment:
Not sure why we need this if we are already waiting for network to be idle

Reply:
it will be safe to keep the waiting on loader as well!